tools/travis/check_changelogs.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -euo pipefail
 
-md5sum -c - <<< "49bc6b1b9ed56c83cceb6674bd97cb34 *html/changelogs/example.yml"
+md5sum -c - <<< "364a16f858a957486eaeb1e12673c39c *html/changelogs/example.yml"
```

Comment:
Where exactly did you get this value? I was looking to mirror to PR.

Reply:
md5sum

Reply:
How does one generate it?

Reply:
md5sum

Reply:
![Like this: md5sum](https://user-images.githubusercontent.com/5211576/65289722-de321400-db19-11e9-953b-fcf307c9bc79.png)

Reply:
![md5sum](https://user-images.githubusercontent.com/5211576/65289988-11c16e00-db1b-11e9-814c-ee39754eb9c3.gif)

Reply:
oh, thank you!